.golangci.yml

```diff
@@ -54,16 +54,19 @@ linters-settings:
       - dot
       - blank
   goheader:
-    values:
     template-path: .libevm-header
 
   gomodguard:
     blocked:
       modules:
         - github.com/ethereum/go-ethereum:
+            reason: ""
```

Comment:
Interesting! How did it know that these fields exist?! I've made suggestions with reasons so my original rationale is now explicit.

Reply:
It didn't, I looked it up because the linter was complaining about the empty object, i.e.

```yml
- blabla:
- abc:
```

is not making the linter happy, so I've set just one field to its zero value 🤷  But good you've added a reason for each, even better 👍